spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -138,16 +138,16 @@
       end
 
       describe "the notes holder" do
-        it { should have_css(".js-temp-notes-holder") }
+        it { is_expected.to have_css(".js-temp-notes-holder") }
 
-        it { within(".js-temp-notes-holder") { should have_css(".new_note") } }
+        it { within(".js-temp-notes-holder") { is_expected.to have_css(".new_note") } }
```

Comment:
Line is too long. [87/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.